bin_tests/src/bin/crashtracker_bin_test.rs

```diff
@@ -13,6 +13,7 @@ fn main() -> anyhow::Result<()> {
 mod unix {
     use anyhow::Context;
     use bin_tests::modes::behavior::get_behavior;
+    use nix::sys::signal::{raise, Signal};
```

Comment:
I don't think it matters, but dropping a note here that the nix `raise()` uses `tgkill()` instead of `kill()` on Linux.  I am not intimately familiar with the implications here.